Common/UnitDefinitions/VolumeConcentration.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Name": "VolumeConcentration",
```

Comment:
Same as for MassFraction, I need to learn why `Ratio` is not sufficient. Keep the discussion in MassFraction thread.